training/distributed_training/tensorflow/data_parallel/mnist/infer_tensorflow.ipynb

```diff
@@ -27,7 +27,7 @@
     "%store -r model_data\n",
```

Comment:
<p>This may help you to pass CI</p><pre class="ql-syntax" spellcheck="false">try:
    model_data
except NameError:
    model_data = 's3://sagemaker-us-west-2-688520471316/tensorflow2-smdataparallel-mnist-2020-10-16-17-15-16-419/output/model.tar.gz'
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1848/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
Thanks for taking a look. Would the CI that's running this notebook have access to following S3 object?
```
s3://sagemaker-us-west-2-688520471316/tensorflow2-smdataparallel-mnist-2020-10-16-17-15-16-419/output/model.tar.gz
```

Reply:
My guess is yes, since I suppose the bucket is public. I did similar thing here https://github.com/aws/amazon-sagemaker-examples/blob/master/frameworks/tensorflow/get_started_mnist_deploy.ipynb 
using a public bucket `sagemaker-sample-files` and CI liked that.

Reply:
Side note: I caught a typo in that notebook. Hence created a PR to fix that: https://github.com/aws/amazon-sagemaker-examples/pull/1871

Reply:
Tried using the above snippet, but seeing some weird unknown infra failures. @hsl89 

Also I don't think that approach of setting the model_data is going to work since that bucket is not public
```
aws s3 ls s3://sagemaker-us-west-2-688520471316/

An error occurred (AccessDenied) when calling the ListObjectsV2 operation: Access Denied
```

Reply:
If the bucket is not public, then this approach of retrieving model artifact is definitely NOT going to work. If you have write access to the public bucket `sagemaker-sample-files` then you can put your model artifact there, else you can give the artifact to me and I will put in the `sagemaker-sample-files`.

Reply:
Updated the model file